tests/unit_tests/test_wave_06.py

```diff
@@ -75,8 +75,13 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
+    assert result == True
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3 
+    assert len(jesse.get_by_category("Clothing")) == 1
+    assert len(jesse.get_by_category("Decor")) == 2
```

Comment:
Good start, but these asserts only look at Jesse's inventory. Also, having the correct number of categories of items does not necessarily mean we have the specific items we are expecting. Instead, it would be good to do something like what you do for `test_swap_best_by_category_reordered()` test